trace-test-application/src/androidTest/java/io/bitrise/trace/testapp/screen/BaseScreen.java

```diff
@@ -65,12 +68,21 @@ public UiObject2 find(@NonNull final BySelector by) {
     }
 
     /**
-     * Performs a click action on the given {@link BySelector}.
+     * Performs a click action on the given {@link BySelector}. Retries the finding 3 times.
```

Comment:
nit: `Retries the finding 3 times` implies that it will happen 4 times (it'll will try the first time, retry once, retry twice, retry the third time). Perhaps `Attempts to find the item 3 times. This is useful because x y z` and explain a bit why it might fail a few times?

Reply:
Updated, thanks for spotting it.